.github/workflows/build.yml

```diff
@@ -8,6 +8,7 @@ on:
   pull_request:
     branches:
       - master
```

Comment:
What exactly does workflow_dispatch do?

Reply:
That's a manual trigger, so you can go to actions tab and trigger a workflow on any branch. I'm not sure it will work for forks, but we'll see (and it doesn't hurt 😅)